bin/phpcs-changed

```diff
@@ -46,6 +46,7 @@ $options = getopt(
 		'cache',
 		'no-cache',
 		'clear-cache',
+		'arc-lint',
```

Comment:
Can we add `--arc-lint` to the README also?

Reply:
Right, added in 57f6d7b